Ability to configure week start day (Monday or Sunday)

rafael-paiva opened the pull request:
I've done some customization to meet my project's requirements, which was to start the week on Sunday instead of Monday. Take a look and see if you want to incorporate these changes. Feel free to make any comments/suggestions :)

willrax commented:
Hey, we released a slight variation on this in the latest 0.5.0 release. Does this cover your use case? I appreciate the work you put in here and thanks for opening the PR. I'll close this for now, but let me know if you think your use case isn't covered and we can reopen.

rafael-paiva commented:
Thanks @willrax!
As soon as I have time I'll try that out and let you know. But by looking at the code I'm pretty sure it covers my scenario.

Thanks again for you feedback 🍻

rafael-paiva commented:
@willrax these changes didn't quite cover my use case because of this issue: https://github.com/alphasights/ember-calendar/issues/128

I've incorporated latest changes to my fork and I'm still using it with a fix for that issue. I also opened a PR for this repo. Let me know if you want to incorporate these changes.

I also discovered another issue: the "THIS WEEK" button isn't disabled when viewing the current week if you have `startFromDate=true`. Also fixed this issue on my PR.

https://github.com/alphasights/ember-calendar/pull/129

Thanks!
